portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,18 +45,36 @@
     * @throws IOException if there is an I/O error
     */
     @Override
-    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
         Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
         List<Comment> comments = new ArrayList<>();
         for (Entity entity : results.asIterable()) {
-            Comment comment = new Comment(entity.getKey().getId(), (String) entity.getProperty("comment"), (long) entity.getProperty("timestamp"));
+            Comment comment = new Comment(entity.getKey().getId(), (String) entity.getProperty("nickname"), (String) entity.getProperty("comment"), (long) entity.getProperty("timestamp"));
```

Comment:
You might want to wrap this long line.  Most auto-formatters for Java (including the ones in most IDEs) will let you specify a wrap column.